controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -95,10 +95,10 @@ type ClowdAppReconciler struct {
 // +kubebuilder:rbac:groups=monitoring.coreos.com,resources=prometheuses;servicemonitors,verbs=get;list;watch;create;update;patch;delete
 
 // Reconcile fn
-func (r *ClowdAppReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+func (r *ClowdAppReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	qualifiedName := fmt.Sprintf("%s:%s", req.Namespace, req.Name)
 	log := r.Log.WithValues("app", qualifiedName).WithValues("id", utils.RandString(5))
-	ctx := context.WithValue(context.Background(), errors.ClowdKey("log"), &log)
+	ctx = context.WithValue(ctx, errors.ClowdKey("log"), &log)
```

Comment:
do we still need to add the logger to the context? I thought they said they added it in the controller-runtime

Reply:
I'm not sure - but this seems to work as expected